datafusion/core/src/datasource/physical_plan/csv.rs

```diff
@@ -1249,4 +1249,20 @@ mod tests {
             }
         }
     }
+
+    /// Get the schema for the aggregate_test_* csv files with an additional filed not present in the files.
+    fn aggr_test_schema_with_missing_col() -> SchemaRef {
```

Comment:
I moved this out as this is the only place it is used and rewrote it to append the column explicitly rather than copy/paste with one line difference